x-pack/filebeat/module/imperva/securesphere/test/generated.log-expected.json

```diff
@@ -0,0 +1,5693 @@
+[
+    {
+        "destination.ip": [
+            "10.70.155.35"
+        ],
+        "destination.port": 892,
+        "event.action": "cancel",
+        "event.category": "Login",
+        "event.code": "Imperva",
+        "event.dataset": "imperva.securesphere",
+        "event.module": "imperva",
+        "event.original": "%IMPERVA-Imperva,dstIP=10.70.155.35,dstPort=892,dbUsername=tatno,srcIP=10.81.122.126,srcPort=4141,creatTime=29 January 2016 06:09:59,srvGroup=uam,service=untutl,appName=rad,event#=taliqu,eventType=Login,usrGroup=ommod,usrAuth=True,application=\"scivel\",osUsername=aqui,srcHost=radipis5408.mail.local,dbName=enatuse,schemaName=magn,bindVar=equuntu,sqlError=failure,respSize=5910,respTime=10.347000,affRows=sum,action=\"cancel\",rawQuery=\"sit\"",
+        "event.outcome": "Success",
+        "fileset.name": "securesphere",
+        "host.hostname": "radipis5408.mail.local",
+        "input.type": "log",
+        "log.offset": 0,
+        "network.application": "scivel",
+        "observer.product": "Secure",
+        "observer.type": "WAF",
+        "observer.vendor": "Imperva",
+        "related.ip": [
+            "10.70.155.35",
+            "10.81.122.126"
+        ],
+        "rsa.counters.dclass_c1": 5910,
+        "rsa.counters.dclass_c1_str": "Affected Rows",
+        "rsa.db.database": "enatuse",
+        "rsa.db.index": "sit",
+        "rsa.internal.messageid": "Imperva",
+        "rsa.investigations.ec_activity": "Logon",
+        "rsa.investigations.ec_outcome": "Success",
+        "rsa.investigations.ec_subject": "User",
+        "rsa.investigations.ec_theme": "Authentication",
+        "rsa.misc.action": [
+            "cancel"
+        ],
+        "rsa.misc.event_type": "Login",
+        "rsa.misc.group": "ommod",
+        "rsa.misc.group_object": "uam",
+        "rsa.misc.result": "failure",
+        "rsa.time.duration_time": 10.347,
+        "rsa.time.starttime": "2016-01-29T08:09:59.000Z",
+        "service.type": "imperva",
+        "source.address": "radipis5408.mail.local",
+        "source.ip": [
+            "10.81.122.126"
+        ],
+        "source.port": 4141,
+        "tags": [
+            "imperva.securesphere",
+            "forwarded"
+        ],
+        "user.name": [
```

Comment:
I'm thinking you probably just want to use the value from the original event that says `osUsername` here. The one for dbUsername could be added to `related.user` though. Also, it looks like the database schema name is getting added for some reason?

Reply:
done. This fixed itself by selecting a single field for `user.name`